mock-services/data/compiler/README

```diff
@@ -0,0 +1,20 @@
+Data for mock compiler service. 
```

Comment:
Minor: rename file from README to README.md even though it currently doesn't use any markup.

Reply:
Done.